src/XRP/shared/rippled-web-socket-schema.ts

```diff
@@ -80,6 +123,70 @@ interface WebSocketTransactionResponse {
   validated: boolean
 }
 
+/**
+ * The standard format(s) for a response from the WebSocket API for an `account_lines` request.
+ * @see https://xrpl.org/account_lines.html
+ */
+type WebSocketAccountLinesResponse =
+  | WebSocketAccountLinesSuccessfulResponse
+  | WebSocketAccountLinesFailureResponse
+
+interface WebSocketAccountLinesSuccessfulResponse {
+  id: number | string
+  status: string
+  type: string
+  result: {
+    account: string
+    ledger_hash: string
+    ledger_index: number
+    lines: Array<TrustLineResponse>
+    validated: boolean
+  }
+}
+
+interface WebSocketAccountLinesFailureResponse {
+  id: number | string
+  status: string
+  type: string
+  error: string
+  error_code: number
+  error_message: string
+  request: AccountLinesRequest
+}
```

Comment:
i'll need to look at the other responses some more, but these seem to share some fields. can we make a base type that they extend? like

```
interface BaseWebSocketAccountLinesResponse {
  id: number | string
  status: string
  type: string
}

interface WebSocketAccountLinesSuccessfulResponse extends BaseWebSocketAccountLinesResponse {
  result: // result thing
}

interface WebSocketAccountLinesFailureResponse extends Base {
  // failure response specific fields
}
```

Reply:
I handle that in #860 but I can move it to here if you think that would be better

Reply:
nah that's cool